src/js/lib/content/src/index.js

```diff
@@ -122,6 +122,101 @@ function show_debug_screen(wasm,msg) {
 
 
 
+// ======================
+// === Crash Handling ===
+// ======================
+
+function recoveringFromCrash() {
+    return sessionStorage.getItem("crash-message") !== null
+}
+
+function previousCrashMessage() {
+    return sessionStorage.getItem("crash-message")
+}
+
+function storeCrashMessage(message) {
+    sessionStorage.setItem("crash-message", message)
+}
+
+function recoveredFromCrash() {
+    sessionStorage.removeItem("crash-message")
+}
+
+
+// === Crash detection ===
+
+function setupCrashDetection() {
+    Error.stackTraceLimit = Infinity
```

Comment:
Even if the IDE runs into an infinite recursion and exceeds the stack size limit, I wouldn't expect more than a few MB of trace data, which still seems okay. But I could also just set the limit to 100 frames or so.

Reply:
I'd stay with 100 frames then. Think about a bug, which would cause massive crashes on many machines. If we've got reported 1k or 10k crashes, each containing several mb, this can be hard to handle.